demisto_sdk/commands/common/xsoar_yaml.py

```diff
@@ -3,29 +3,29 @@
 
 class XSOAR_YAML:
     def __init__(self, preserve_quotes=True, allow_duplicate_keys=True, default_flow_style=None, width=None, typ=None):
-        self.xsoar_yaml = YAML()
-        self.xsoar_yaml.preserve_quotes = preserve_quotes
-        self.xsoar_yaml.allow_duplicate_keys = allow_duplicate_keys
-        self.xsoar_yaml.default_flow_style = default_flow_style
-        self.xsoar_yaml.width = width
-        self.xsoar_yaml.typ = typ
+        self.ryaml = YAML()
```

Comment:
Perhaps name it `_ryaml`, discouraging anyone from accessing `xsoar_yaml.ryaml.command_or_attribute`, and allowing easier maintenance?

This will make customizing things much harder -- for better or worse.
wdyt? @ilaner @jochman